### PR TITLE
Remove of Iframes - projects have now URLs (was: now all the admins can be seen)

### PR DESCRIPTION
**Pull request by dpshelio**
~A bit hacky... but this removes references to the years and allows multiple years for org-admins~

So this is the so awaited feature by @Cadair!!! Now you can share your projects ideas with your friends!!

There's so many files added because I needed to use sass before I was becoming crazy!! All there!! Anyone dares to try it out??

### CODE REVIEW

File: gsoc/display/partials/tabs/mentors.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
             <div class="title-get-involved" >Admins</div>
             <br>
             <div class="description">
-                We are the admins for OpenAstronomy in 2017.
```
Comment (bsipocz) [inline gsoc/display/partials/tabs/mentors.html]: you could maybe use the js to fill in the year, too?

**Comment by dpshelio** [comment]
Just notice that everything in the new version is working, however now we look closer to each other!!!

With the iframe
oa_asitwas

With this new PR
oa_asitsnow

I'm sure it's possible to bring that space between all of us back again (also for the ideas)... However, I've spent too much time with it.

**Comment by dpshelio** [comment]
I've now added bootstrap as a submodule. I cannot do the same with materialize because I needed to modify it (only one line!! :( )

**Comment by Cadair** [comment]
@dpshelio you utter nutter! Awesome!

I have a couple of high-level points, but it looks fine to me on my local machine.

1. The font in the projects section is not consistent with the rest of the site and is serif. I tried to work out where to change it but I couldn't find it.
2. The % height of the project idea pop up seems low to me, would be better if it was taller.

Other than that, fantastic!!

**Comment by dpshelio** [comment]
Right now is 50% height the screen (25% above and 25% below). Let me know which one seems good to you.

Regarding the font... I tried to keep the original one from that thingy... but I'm happy to change it if you think of anyone better.

**Comment by bsipocz** [comment]
@dpshelio - How does the years work? In the screenshots above both have 2017, and at least the top one should read 2018...

**Comment by dpshelio** [comment]
The picture are from the 2017 season, because I had more people on these.

**Comment by Cadair** [comment]
> Right now is 50% height the screen (25% above and 25% below). Let me know which one seems good to you.

I would make it maybe 10% either side, there is a lot of content in that popout and it should take up most of the screen in my opinion.

> Regarding the font... I tried to keep the original one from that thingy... but I'm happy to change it if you think of anyone better.

I would see how it looks with the same one we use in thr main page content, it's really weird to me to have it be inconsistent.

**Comment by Cadair** [comment]
Actually on the font thing, it's not the same as it is on the current live page. The current live page has a sans-seif font, which is not as visually different from the rest of the page. I would still rather it all be consistent though.

**Comment by Cadair** [comment]
for context:

The menu bar (on the live page) has: `font-family: Helvetica, Arial, sans-serif;` and the ideas page has `font-family: "Roboto", sans-serif;`

I don't mind which way we go with this, but I would pick one and make the whole site use it.

**Comment by bsipocz** [comment]
@dpshelio - Actually the year is also bad on the current mentor page, it says 2017 (and the first picture if from there, as we have one mentor in that picture who is new to mentoring this year ;) )

**Comment by dpshelio** [comment]
You are correct @bsipocz and that confused me when I took the screenshot. But my point was that the pictures are too closed together in this new one... and that the font is slightly different (at least on the title)

**Comment by bsipocz** [comment]
yeah, I know my issue is orthogonal, but one that needs to be addressed before out submission

**Comment by dpshelio** [comment]
@Cadair - yes, the projects page ships with Roboto. We can take it out.

**Comment by dpshelio** [comment]
@bsipocz - the dates is fixed in this PR anyway :)

**Comment by Cadair** [comment]
@dpshelio I wonder then if it's because I don't have roboto installed. Which leads me to wonder a) why it's falling back to a serif font and b) why it's not downloading roboto from google web fonts?

File: _layouts/projects.html [hunk 2]
```diff
@@ -27,60 +27,42 @@
...
-    <script type="text/javascript" src="./vendors/angular-sanitize/angular-sanitize.js"></script>
-    <script type="text/javascript" src="./vendors/showdown/showdown.js"></script>
-    <script type="text/javascript" src="./vendors/markdown/markdown.js"></script>
-    <script type="text/javascript" src="./resources/js/app.js"></script>
-
-    <link rel="stylesheet" type="text/css" href="./vendors/bootstrap/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="./vendors/materialize/css/materialize.css">
-    <link rel="stylesheet" type="text/css" href="./resources/css/style.css">
-
-    <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:300,300i,400,400i,700,700i|Roboto:100,200,300,400,500,600,700,800,900|Ubuntu+Mono|Overpass+Mono|Inconsolata" rel="stylesheet">
```
Comment (Cadair) [inline _layouts/projects.html]: I think this is part of the problem, this stylesheet needs to be in the page somewhere to download roboto if needed.

**Comment by Cadair** [comment]
you only live once...

**Pull request merged**